fauna/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: Fauna
+
+## Overview
+
+This integration allows Fauna users to import their database query logs into Datadog. Users are able to configure it for specific databases or for entire region groups. When enabled all requests made to fauna matching the database or region group configuration will show up in Datadog logs under the fauna service.
+To see what fields are included, see the [Fauna Query Log Record Format][1].
+
+## Setup
+
+### Installation
+
+Use the 'Connect Accounts' button above to connect your Fauna and Datadog account. This will allow Fauna to send logs to your Datadog account and will take you through a brief configuration process.
```

Comment:
This should include each step for set up the integration including each OAuth flow step